Runtime/SimvaPlugin.cs

```diff
@@ -159,18 +189,15 @@ private void DestroyPreviousSimvaScene()
         public void StartGameplay()
         {
             DestroyPreviousSimvaScene();
-            Debug.Log("Starting Gameplay");
-            if (SceneManager.GetActiveScene().name  != GamePlayScene)
-            {
-                SceneManager.LoadScene(GamePlayScene);
-            }
+            Log("Starting Gameplay");
+            RunScene("gameplay");
```

Comment:
Should use uppercase as naming convention

Reply:
fixed